client/src/iiq-commands.ts

```diff
@@ -13,10 +13,11 @@ var xpath = require('xpath')
 import { DOMParser } from 'xmldom'
 import { XMLSerializer } from 'xmldom'
 import { API as GitAPI, Repository, GitExtension, Status } from './typings/git';
+import { PathProposer } from './pathProposer';
 
 const fg = require('fast-glob');
 
-export namespace Commands{
```

Comment:
When you start modifying someone else's code you can't format it to your liking all over the place, please get back to the old formatting.

Reply:
I agree. But this needs to be defined somewhere.
I just used the "format document" command of VSCode.

Reply:
I will look into https://editorconfig.org/ to align with your code style and propose a new `.editorconfig` file

Reply:
I spent more than 1 hour trying to replicate the formatting style of `iiq-commands.ts` with a combination of VSCode settings, ESLint and/or prettier. It does not work for some reason.
The thing is that the formatting style is not consistent between `iiq-commands.ts` and `extension.ts` regarding the spaces before and after brackets for blocks (functions, if, etc.) and for objects.
If you are not happy with the default formatting style of VSCode, you should define it in either way you like (ESLint, prettier, etc.) and enforce it in the settings, as you have pushed them in the repository.

Reply:
No editor should impose its formatting style, otherwise people would override each other's formatting back and forth when contributing to each other's projects.
I am totally fine with your coding preferences when you create your code though (just as I am free to convert it to my style later).